Dockerfile

```diff
@@ -9,4 +9,4 @@ ADD tests ./tests
 
 ADD setup.py ./
 
-RUN python setup.py pytest
```

Comment:
Otherwise a broken test was breaking the build and wasn't visible in the Tests panel
<img width="1176" alt="Screenshot 2023-11-10 at 15 08 16" src="https://github.com/spreaker/python-lambda-powertools/assets/4105325/5c39e0c1-a54a-43ec-958a-9feb35c911e3">